src/js/wall/Queries.ts

```diff
@@ -0,0 +1,30 @@
+import { login } from "@/account/api";
+import { ErrorResponse } from "@/types/types";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { Response } from "superagent";
+
+/**
+ * Factory object for generating account query keys
+ */
+export const accountKeys = {
+    all: () => ["account"],
+    details: () => ["account", "details"] as const,
+};
```

Comment:
Better to import the pre-existing `accountKeys` factory function from `account/queries`. Prevents potential drift that could occur over time